frontend/src/routes/(authenticated)/org/[org_id]/+page.ts

```diff
@@ -141,6 +144,9 @@ export async function _addOrgMember(orgId: UUID, emailOrUsername: string, role:
       `),
       { input: { orgId, emailOrUsername, role, canInvite} },
     );
+  if (withProjects && withProjects.length > 0) {
+    await _addProjectsToOrg({ orgId, projectIds: [...withProjects.map(p => p.id)]})
```

Comment:
rather than making this a seperate api call, you can actually do multiple mutations in one reqeust. this mutation query could be modified to look like this:
```gql
mutation AddOrgMember($inputMember: SetOrgMemberRoleInput!, $inputProjects: AddProjectsToOrgInput!) {
  setOrgMemberRole(input: $inputMember) {
    organization {
      id
    }
    errors {
      __typename
      ... on Error {
        message
      }
    }
  }
  addProjectsToOrg(input: $inputProjects) {
    organization {
      id
    }
    errors {
      __typename
      ... on Error {
        message
      }
    }
  }
}
```
as an example, you would probably want to make the `addProjectsToOrg` mutation return early if the array was empty but that's simple enough

Reply:
Done.